doc/xl2tpd-control.8

```diff
@@ -1,10 +1,10 @@
-.TH xl2tpd-control 8  "Sep 2020"
+.TH xl2tpd-control 8 "Jan 2022"
 
 .SH NAME
 xl2tpd\-control \- xl2tpd control utility.
 
 .SH DESCRIPTION
-xl2tpd is an implementation of the Layer 2 Tunneling Protocol (RFC 2661).
+xl2tpd is a FREE implementation of the Layer 2 Tunneling Protocol (RFC 2661).
```

Comment:
Could you go into why you are adding  "FREE". Is there a reason you wan to add it?

Reply:
just want to pay attention that Xelerance working on open implementation of L2TP, i.e. xl2tpd is a Free Software

Reply:
Thank you for the explanation.